### ISSUE TITLE
Update the Electron to latest version

### ISSUE THREAD

**Issue by ms10398** [fossasia/susi_desktop#105]
**Actual Behaviour**

Electron version is not updated the version 1.7.8 has security flaws so the electron version should be updated to 1.8.2

**Expected Behaviour**

Electron version should be the latest version

##### Please check the applicable fields -

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [x] Assign this issue to me (I want to work on this).
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

I will update it to the latest version in my PR.

In future we can look for implementing automatic dependecies update.
